docs/development/testguide.rst

```diff
@@ -304,6 +414,9 @@ Examples
 .. code-block:: none
 
     from ...config import get_data_filename
+    # Note, this is retained for backwards compatibility but the remote_data
+    # decorator is now provided by the pytest-remotedata plugin, and does not
+    # need to be imported from astropy.
     from ...tests.helper import remote_data
```

Comment:
I would totally remove this from the docs. We provide the backward compatibility, but the narrative docs should only contain the recommended way of usage.